geode-lucene/geode-lucene-test/src/main/java/org/apache/geode/cache/lucene/test/SpecialCustomer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.cache.lucene.test;
+
+import java.util.Collection;
+
+public class SpecialCustomer extends LocalCustomer {
```

Comment:
I find the names "LocalCustomer" and "SpecialCustomer" to not be very descriptive. Maybe InheritedCustomer or ChildCustomer or something like that which describes what you are trying to do with your test?

Reply:
I was using ChildCustomer before, then GrandChildCustomer. But I think LocalCustomer and SpecialCustomer makes more sense in naming. How do you think?